felles/src/test/kotlin/no/nav/helsearbeidsgiver/felles/rapidsrivers/service/ServiceRiverStatefulTest.kt

```diff
@@ -45,33 +44,12 @@ class ServiceRiverStatefulTest :
         context("fail-melding har presedens") {
             withData(
                 mapOf(
-                    "over start" to
-                        mapOf(
-                            Key.EVENT_NAME to mockService.eventName.toJson(),
-                            Key.UUID to UUID.randomUUID().toJson(),
-                            Key.FAIL to Mock.fail.toJson(Fail.serializer()),
-                            Key.DATA to "".toJson(),
-                        ).plus(Mock.values(mockService.startKeys)),
                     "over data" to
                         mapOf(
                             Key.EVENT_NAME to mockService.eventName.toJson(),
                             Key.UUID to UUID.randomUUID().toJson(),
                             Key.FAIL to Mock.fail.toJson(Fail.serializer()),
-                            Key.DATA to "".toJson(),
-                        ).plus(Mock.values(mockService.dataKeys)),
-                    "over start med nested data" to
-                        mapOf(
-                            Key.EVENT_NAME to mockService.eventName.toJson(),
-                            Key.UUID to UUID.randomUUID().toJson(),
-                            Key.FAIL to Mock.fail.toJson(Fail.serializer()),
-                            Key.DATA to Mock.values(mockService.startKeys).toJson(),
-                        ),
-                    "over nested data" to
-                        mapOf(
-                            Key.EVENT_NAME to mockService.eventName.toJson(),
-                            Key.UUID to UUID.randomUUID().toJson(),
-                            Key.FAIL to Mock.fail.toJson(Fail.serializer()),
-                            Key.DATA to Mock.values(mockService.dataKeys).toJson(),
+                            Key.DATA to mockService.mockSteg1Data().toJson(),
```

Comment:
Er det sånn at vi ikke trenger noen test som sjekker det caset for startdata (Steg0Data) nå, fordi det oppfører seg i praksis helt likt som steg1 nå som alt av data ligger nøstet inne i data-feltet?

Reply:
Stemmer! Før så hadde vi tre typer meldinger: start, data og fail. Nå har vi merget start og data til én, siden de begge egentlig bare bar data.